openedx/core/djangoapps/content/block_structure/config/__init__.py

```diff
@@ -16,9 +16,9 @@
 # .. toggle_description: When enabled, the block structure cache is invalidated when changes to
 #   courses are published. If `block_structure.storage_backing_for_cache` is active, all block
 #   structures related to the published course are also cleared from storage.
-# .. toggle_warnings: This switch is deprecated and will be removed at some point.
-#   It is recommended to set this to `True`, which will be the new default after deprecation.
-#   For more information, check https://github.com/edx/edx-platform/pull/26175#issuecomment-769080286
+# .. toggle_warnings: This switch will likely be deprecated and removed.
+#   It is recommended to keep this as `False`, which will be the new default after deprecation.
```

Comment:
Yikes.  It looks like most of these switch to True except this one?  Would you mind replying with the appropriate PR comments if you want me to double-check the recommendations?

Reply:
Definitely 😄 This should be all:
- `block_structure.raise_error_when_not_found`: [`True`](https://github.com/edx/edx-platform/pull/26175#discussion_r565796224)
- `block_structure.invalidate_cache_on_publish`: [`True`](https://github.com/edx/edx-platform/pull/26175#discussion_r565800087)
- `block_structure.storage_backing_for_cache`: [`False`](https://github.com/edx/edx-platform/pull/26175#discussion_r565801078)
- `BLOCK_STRUCTURES_SETTINGS['PRUNING_ACTIVE']` : [`True`](https://github.com/edx/edx-platform/pull/26175#issuecomment-769211933)

Reply:
Thanks for those links. That helped a lot. The only one that I read differently is Nimisha's comment about `raise_error_when_not_found`. It sounded to me like she was voting for removing it and leaving it off.

If you aren't sure which way to go, we could drop the recommendation like and ask the question in the DEPR ticket about which way we should go.

Reply:
@robrap I'm glad the links helped 😄 

Regarding `raise_error_when_not_found`, I think it would be better to refer to this https://github.com/edx/edx-platform/pull/14512/files#r103130819 instead. This comment, on the original pull request, states that they want errors raised. cc @nasthagiri 

It's a little bit confusing, so I guess we might as well drop the recommendation line, what are your thoughts on that?

Reply:
Thanks @nizarmah.  I'm going to merge this.  You can discuss and update recommendations when you update to point to the DEPR ticket.

I think that @nasthagiri was saying that since `raise_error_when_not_found` was never turned on for us, it may have either had an issue, or it was forgotten about and hasn't been an issue. We want to remove as many flags as possible.  Removing this as True would mean we'd first need to prove its worthiness, since it was never enabled for us.  Removing it as False would mean no change for edX.org, but I don't know about the external community, and they are the types of things that can be discussed in the DEPR ticket. Thank you!